puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/plugins.groovy

```diff
@@ -21,6 +21,12 @@ pipeline {
             steps {
                 // TODO: from variables
                 repoclosure('plugins', 'el7', foreman_version)
+
+                script {
+                    if (foreman_version != '2.0' || foreman_version != '1.24' || foreman_version != '1.23') {
```

Comment:
Rather than hardcoding it here, have you considered the TODO with variables?

Reply:
I have, except, I don't know what that TODO means :)

Reply:
In https://github.com/theforeman/foreman-infra/tree/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/foreman we have variables for every release. The idea was to define the EL versions there. That prepares you for dropping EL7 as well. I didn't do that when I wrote this because we only had one EL version.

Reply:
Now I gotcha, updated